src/isar/mission_planner/echo_planner.py

```diff
@@ -172,3 +150,8 @@ def _create_thermal_image_task(self, tag_name: str) -> TakeThermalImage:
         )
 
         return thermal_image_task
+
+
+class EchoInspectionNames(Enum):
+    TakeThermalImage: str = "ThermicPicture"
+    TakeImage: str = "Picture"
```

Comment:
I think it is cleaner to make it a frozen (immutable) dataclass
```suggestion

@dataclass(frozen=True)
class EchoInspectionNames():
    TakeThermalImage: str = "ThermicPicture"
    TakeImage: str = "Picture"
```